docs/docs/how_to/index.mdx

```diff
@@ -83,6 +83,7 @@ These are the core building blocks you can use when building applications.
 - [How to: track response metadata across providers](/docs/how_to/response_metadata)
 - [How to: use chat model to call tools](/docs/how_to/tool_calling)
 - [How to: stream tool calls](/docs/how_to/tool_streaming)
+- [How to: rate limit chat models](/docs/how_to/chat_model_rate_limiting)
```

Comment:
makes it sound like you're adding rate limiting, vs matching the api's rate limiting. maybe: "How to handle rate limits"?